lib/runtime/src/storage/key_value_store/etcd.rs

```diff
@@ -218,11 +218,11 @@ impl EtcdBucket {
         }
         let current_version = kvs.first().unwrap().version() as u64;
         if current_version != version + 1 {
-            tracing::warn!(
+            tracing::debug!(
```

Comment:
`debug` implies it's fine. Is it? Doesn't it imply we either sent the wrong revision, or our update got clobbered?